k8Config/emailtodb-deployment.yaml

```
@@ -33,4 +86,23 @@ spec:
+        - name: outlookUserEmail
+          value: ${OUTLOOK_USER_EMAIL}
+        - name: outlookConnectionTimeout
+          value: "${OUTLOOK_CONNECTION_TIMEOUT:30}"
+        - name: outlookReadTimeout
+          value: "${OUTLOOK_READ_TIMEOUT:60}"
+        - name: outlookRetryMaxAttempts
+          value: "${OUTLOOK_RETRY_MAX_ATTEMPTS:3}"
+        - name: emailProvidersEnabled
+          value: "${EMAIL_PROVIDERS_ENABLED:gmail,outlook-enhanced}"
+        - name: multiProviderEnabled
+          value: "${MULTI_PROVIDER_ENABLED:true}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)